Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,232 @@
+                        if (holderCount == 1)
+                        {
+                            highestBalanceHolder += ($", {currentBalanceHolder}");
+                            holderCount++;
+                        }
+                    }
+                    highestBalance = highestBalance < balance ? balance : highestBalance;
+                }
+            }
+            string stringHighestBalance = highestBalance.ToString();
+            return ($"{highestBalanceHolder} had the most money ever. ¤{stringHighestBalance}.");
         }
 
         /// <summary>
         /// Return name and loss of a person with a biggest loss (balance change negative).
         /// </summary>
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
-        }
+            double biggestLoss = 0f;
+            double balance;
+            double nextBalance;
+            string biggestLoser = "";
+            int holderCount = 0;
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+            foreach (string entry in peopleAndBalances)
+            {
+                string[] userHistory = entry.Split(',');
+                if (userHistory.Length <= 2)
+                {
+                    return "N/A.";
+                }
+                string currentBalanceHolder = userHistory[0];
+
+                for (int i = 1; i < userHistory.Length - 1; i++)
+                {
+                    balance = double.TryParse(userHistory[i], out double result) ?
+                        double.Parse(userHistory[i]) : double.NaN;
+
+                    nextBalance = double.TryParse(userHistory[i + 1], out double nextResult) ?
+                        double.Parse(userHistory[i + 1]) : double.NaN;
 
+                    if ((nextBalance - balance) < biggestLoss)
+                    {
+                        biggestLoser = currentBalanceHolder;
+                        holderCount = 1;
+                    }
+                    if (biggestLoss == nextBalance - balance)
+                    {
+                        if (holderCount == 1)
+                        {
+                            biggestLoser += ($", {currentBalanceHolder}");
+                            holderCount++;
+                        }
+                        if (biggestLoss == nextBalance - balance &&
+                            biggestLoser != currentBalanceHolder &&
+                            holderCount > 1)
+                        {
+                            biggestLoser += ($" and {currentBalanceHolder}");
+                        }
+                    }
+                    biggestLoss = ((nextBalance - balance) < biggestLoss) ?
+                        (nextBalance - balance) : biggestLoss;
+                }
+            }
+            string stringBiggestLoss = Math.Abs(biggestLoss).ToString();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4953253)